0-bootstrap/versions.tf

```diff
@@ -31,4 +31,9 @@ terraform {
       version = "~> 2.3"
     }
   }
+
+  provider_meta "google" {
+    module_name = "blueprints/terraform/terraform-example-foundation:bootstrap/v1.0.0"
```

Comment:
should this now be v2.0.0?

Reply:
@rjerrems release-please should autoupdate this